src/common/playlists/PlayList.jsx

```diff
@@ -15,6 +15,7 @@ const PlayList = () => {
 
   let { playid } = useParams(); // return the parameter of url
   useEffect(() => {
+
```

Comment:
I recommend removing this empty useEffect.

Reply:
done